hie-apply-refact/Haskell/Ide/ApplyRefactPlugin.hs

```diff
@@ -1,9 +1,10 @@
 {-# OPTIONS_GHC -fno-warn-partial-type-signatures #-}
 
-{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE CPP                   #-}
```

Comment:
Is there a reason why you changed the alignment of a lot of pragmas? I don’t have any strong feelings about it but it should be consistent. One advantage of the previous style is that adding a pragma doesn’t potentially require modifications to all other pragmas which can clutter diffs. Also the previous style conforms to what hindent does which seems to become a popular style so it might make sense to not diverge to far from it.